Updating docs to reflect the recent changes (#14)
piiano/vault-python: sdk/orm-django/README.md

   - `vault_collection` (**optional**) - The name of the vault collection that this field is related to. Defaults to `settings.VAULT_DEFAULT_COLLECTION`
   - `vault_property` (**optional**) - The name of the property in the vault collection that this field is related to. Defaults to the name of the field in django.
   - `data_type_name` (**optional**) - The name of the data type in vault. Defaults to 'string'. This only has impact when generating a vault migration, and does not change the way your django model would behave.
   - `eager` (default: **true**) - whether or not value will be decrypted (in a batch operation) as soon as it is fetched from the DB. If not, the value will be decrypted the first time it is accessed.

   **Note**: use `vault_collection` together with `vault_property` to specify the collection and property in vault that represent this field. This is important for permission control and audit logs. For more advanced use-cases, this would allow you to transition smoothly to using Vault as a secure storage for PII data.

Query your model as usual, keeping the following in mind:

> * Read queries are batched, so reading from the DB will generate one API call per field, while writing to the DB is not batched (so an API call for each field in each instance).

Reviewer: ```suggestion
* Read queries are batched. Reading from the Database will generate a single API call per field. Writing to the Database is not batched and will generate an API call for each field in each instance.
```

> * By default all fields are eager - similarly to calling prefetch_related(field_name) on a foreign key.

Reviewer: ```suggestion
* By default all fields are eagerly fetched - similarly to calling prefetch_related(field_name) on a foreign key.
```
